core/core.mk

```diff
@@ -29,12 +29,19 @@ cppflags$(sm)	+= -Ilib/libutee/include
 # configuration file
 
 conf-file := $(out-dir)/core/include/generated/conf.h
+conf-in-file := $(out-dir)/conf.in
```

Comment:
What does the .in extension stand for?
I would prefer `conf.mk` or `config.mk` to make it clear that it's a file parsed by make.